CHANGELOG.md

```diff
@@ -1,8 +1,16 @@
 # Changelog
 
-## 1.24.0-dev
+## 1.24.0
 * Feature - Configurable poll duration for container stats [@jcbowman](https://github.com/jcbowman) [#1646](https://github.com/aws/amazon-ecs-agent/pull/1646)
 * Feature - Add support to remove containers and images that are not part of ECS tasks [#1752](https://github.com/aws/amazon-ecs-agent/pull/1752)
+* Feature - Add ARM build script [#1747](https://github.com/aws/amazon-ecs-agent/pull/1747)
```

Comment:
I do not think we need to add this one. ARM build script is for our own process.

Reply:
Make sense, will remove this